apps/spark/src/main/java/com/linkedin/openhouse/jobs/spark/SnapshotsExpirationSparkApp.java

```diff
@@ -78,7 +73,7 @@ public static void main(String[] args) {
             cmdLine.getOptionValue("tableName"),
             Integer.parseInt(cmdLine.getOptionValue("maxAge", "0")),
             cmdLine.getOptionValue("granularity", ""),
-            Integer.parseInt(cmdLine.getOptionValue("minVersions", "0")));
```

Comment:
is this the bug?

Reply:
Yup it was reading minVersions instead of versions, effectively not using the versions configuration. We renamed `minVersions` to `versions` but I missed this line.